config/default.ts

```diff
@@ -20,7 +20,7 @@ export default {
   },
 
   userAccessToken: {
-    cookieName: `rcal-session`,
+    cookieName: `rcal-session-${NODE_ENV}`,
```

Comment:
suggestion: we should show `rcal-session` for prod and `rcal-session-staging` for staging.

Reply:
Done